ports/stm32/boards/stm32g0xx_hal_conf_base.h

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
please make this file look more like `ports/stm32/boards/stm32g4xx_hal_conf_base.h` in terms of style/formatting